facebook/__init__.py

```diff
@@ -94,6 +95,14 @@ def __init__(
         self.timeout = timeout
         self.proxies = proxies
         self.session = session or requests.Session()
+        self.app_secret_hmac = None
+
+        if app_secret and self.access_token:
+            # Generates an app secret hmac based on
+            # https://developers.facebook.com/docs/graph-api/securing-requests
+            self.app_secret_hmac = create_appsecret_proof(
```

Comment:
Since the `create_appsecret_proof` method is only called from this one location, does it need to be a separate method?